Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -976,6 +978,36 @@ public boolean areDeathMessagesEnabled() {
         return config.getBoolean("death-messages", true);
     }
 
+    private String vanishingItemPolicy;
+
+    public String _getVanishingItemsPolicy() {
+        String value = config.getString("vanishing-items-policy", "keep").toLowerCase(Locale.ENGLISH);
+        if (!value.equals("delete") && !value.equals("drop") && !value.equals("keep")) {
+            value = "delete";
```

Comment:
Isn't the default supposed to be "keep"?

Reply:
Yes smh